pkgs/top-level/python-packages.nix

```diff
@@ -8622,7 +8622,7 @@ in {
       sha256 = "1z27wdxs5rj5xhhqfzvzn3yg682irkxw6dcs5jj7mcf97psk8gd8";
     };
 
-    buildInputs = with self; [ nose pybcrypt];
+    propagatedBuildInputs = with self; [ nose bcrypt ];
```

Comment:
Is there a reason why you moved it from `buildInputs` to `propagatedBuildInputs` ?